Screenbox/Pages/MusicPage.xaml.cs

```diff
@@ -55,7 +60,20 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
             }
             else
             {
-                LibraryNavView.SelectedItem = LibraryNavView.MenuItems[0];
+                muxc.NavigationViewItem? libraryNavItem = null;
+                var launchPage = Settings.LaunchPage;
+                if (launchPage.GetNavPageFirstLevel() != "music")
+                {
+                    libraryNavItem = (muxc.NavigationViewItem?)LibraryNavView.MenuItems[0];
+                }
+                else
+                {
+                    var launchPageTag = launchPage.GetNavPageSecondLevel();
+                    libraryNavItem = LibraryNavView.MenuItems
+                        .OfType<muxc.NavigationViewItem>()
+                        .FirstOrDefault(item => item.Tag?.ToString() == launchPageTag);
+                }
```

Comment:
With this flow, the second-level page is set each time the user navigates to the parent page. Is this expected? The settings text suggests that this behaviour only applies to app launches. However, in reality, it is set on every page navigation.

Reply:
From my testing, it actually only applies to app launches, not every page navigation.